benchmark/storage/garbage_collector_benchmark.cpp

```diff
@@ -150,6 +150,33 @@ BENCHMARK_DEFINE_F(GarbageCollectorBenchmark, HighContention)(benchmark::State &
   state.SetItemsProcessed(state.iterations() * num_txns - lag_count);
 }
 
+/**
+ * Run a large number of updates on a table to generate long version chains. Measures the overhead of garbage collection
+ * in long version chains.
+ * Measure the number of transactions that the GC managed to free during the workload by subtracting the number of
+ * "lagging" transactions that still remained to be cleaned up by the GC after the workload was done running.
+ */
+// NOLINTNEXTLINE
+BENCHMARK_DEFINE_F(GarbageCollectorBenchmark, LongRunningOLAP)(benchmark::State &state) {
+  uint64_t lag_count = 0;
+  // NOLINTNEXTLINE
+  for (auto _ : state) {
+    LargeTransactionBenchmarkObject tested({8, 8, 8}, initial_table_size, 100, update_select_ratio, &block_store_,
+                                           &buffer_pool_, &generator_, true);
+    StartGC(tested.GetTxnManager());
+    auto *olap_txn = tested.GetTxnManager()->BeginTransaction();
+    uint64_t elapsed_ms;
+    {
+      common::ScopedTimer timer(&elapsed_ms);
+      tested.SimulateOltp(num_txns, num_concurrent_txns);
```

Comment:
Why use SimulateOltp() function for olap?

Reply:
There is one OLAP transaction `olap_txn` which starts before all the other OLTP transactions, and would run concurrently with all of them. The `SimulateOltp(..)` function simulates the OLTP transactions.